src/analytics/redux/tracked_events.js

```diff
@@ -19,6 +69,41 @@ function getBlockObject( block ) {
 	return select( 'core/block-editor' ).getBlock( block ) || {};
 }
 
+/**
+ * Track block replacement.
+ *
+ * @param {Array}          originalBlockIds ids or blocks that are being replaced
+ * @param {Object | Array} blocks           block instance object or an array of such objects
+ * @param {Array}          args             additional insertBlocks data e.g. metadata containing inserter method.
+ * @return {void}
+ */
+const trackBlockReplacement = ( originalBlockIds, blocks, ...args ) => {
+	const metaData = args?.[ 3 ] ?? {};
```

Comment:
Similar to [this comment](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976#discussion_r1267781815), we could consider defining the arguments coming from `replaceBlock`/`replaceBlocks`.

Reply:
Good idea! Updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976/commits/7277128f7526d0557303f58a89bca48120c3f88e